src/index.html

```diff
@@ -7,49 +7,36 @@
+<div class="panel open-banner panel-black panel-no-border padding-leader-1 padding-trailer-2">
+  <div class="grid-container">
+      <div class="column-11 pre-1 padding-leader-2 padding-trailer-2">
+        <h1 class="font-size-6 text-white">Esri Open Source</h1>
+        <p class="text-white font-size-1">Thinking spatially, solving problems, together.</p>
+        <a href="https://github.com/esri" class="btn btn-clear-white">Explore 400+ projects on GitHub</a>
       </div>
-      <h1 id="about" class="trailer-half">Esri <a class="link-white" href="http://www.esri.com/software/open/open-source">Open Source</a></h1>
-      <p class="font-size-1">
-        At Esri we believe that spatial thinking can address the world's most difficult problems and enable communities to work together. Central to our mission is the development of advanced technology that enables you to be innovative by providing a stable data management and analysis platform that is built and extended through open&#8209;source code.
-      </p>
-      <p class="font-size-1">
-        These projects cover the entire ArcGIS platform, from user applications to data analysis and visualization. Learn more about our many open-source products and tools that you are free to use, modify and contribute to. We look forward to working together.
-      </p>
-      <p class="home-subscription-text">
-        Need an <a href="//developers.esri.com/sign-up/" class="link-light-blue">ArcGIS subscription</a>? Start developing today for <a href="//developers.arcgis.com/plans/" class="link-light-blue">free</a>.
-      </p>
-    </div>
-  </main>
+  </div>
 </div>
-<div id="wrapper">
-  <!--div id="explore" class="grid-container">
-    <div class="filter-container column-18 pre-3 post-3 leader-0 trailer-1">
-      <div class="label">
-        <em class="lable-it">Find projects by language or keyword:</em>
+
+<div class="panel panel-white panel-border-bottom">
+  <div class="column-16 pre-4 post-4">
+    <div class="block-group block-group-4-up phone-block-group-1-up text-center">
+      <div class="block phone-trailer-1">
+        <h5 class="trailer-0"><a href="#webdevelopment" class="btn btn-large btn-white">Web Development</a></h5>
       </div>
-      <div class="search-select">
-        <select data-placeholder="Search/Select" class="chzn-select" multiple tabindex="6">
-          <optgroup label="Language">
-          {{ languageOptions(data.searchTags.languages) }}
-          </optgroup>
-          <optgroup label="Tags">
-          {{ languageOptions(data.searchTags.tags) }}
-          </optgroup>
-        </select>
+      <div class="block phone-trailer-1">
+        <h5 class="trailer-0"><a href="#datamanagement" class="btn btn-large btn-white">Data Management</a></h5>
+      </div>
+      <div class="block phone-trailer-1">
+        <h5 class="trailer-0"><a href="#spatialanalysis" class="btn btn-large btn-white">Spatial Analyses</a></h5>
```

Comment:
src/assets/img should definitely work, but if you add a new img you have to restart `npm start` because the images are only copied at start up. I'll see about fixing that issue with the file watcher.